automation/src/test/scala/org/broadinstitute/dsde/test/api/WorkspacesAzureApiSpec.scala

```diff
@@ -32,25 +37,116 @@ class AzureWorkspacesSpec extends AnyFlatSpec with Matchers with CleanUp {
     Some(UUID.fromString("f41c1a97-179b-4a18-9615-5214d79ba600"))
   )
 
-  implicit val token: AuthToken = ownerAuthToken
+  private val wsmUrl = RawlsConfig.wsmUrl
+
+  implicit val system = ActorSystem()
 
   "Rawls" should "allow creation and deletion of azure workspaces" in {
+    implicit val token = owner.makeAuthToken()
+    withTemporaryAzureBillingProject(azureManagedAppCoordinates) { projectName =>
+      val workspaceName = generateWorkspaceName()
+      Rawls.workspaces.create(
+        projectName,
+        workspaceName,
+        Set.empty,
+        Map("disableAutomaticAppCreation" -> "true")
+      )
+      try {
+        val response = workspaceResponse(Rawls.workspaces.getWorkspaceDetails(projectName, workspaceName))
+        response.workspace.name should be(workspaceName)
+        response.workspace.cloudPlatform should be(Some(WorkspaceCloudPlatform.Azure))
+        response.workspace.workspaceType should be(Some(WorkspaceType.McWorkspace))
+      } finally {
+        Rawls.workspaces.delete(projectName, workspaceName)
+        assertNoAccessToWorkspace(projectName, workspaceName)
+      }
+    }
+  }
+
+  it should "allow access to WorkspaceManager API" in {
```

Comment:
Similar to other test files, created an explicit test point for verifying that we have access to WSM API.